### PR TITLE
Feature/6 25 2023/property info

### PR DESCRIPTION
**Pull request by prochessor**

### CODE REVIEW

File: models/propertyInfo.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    size: Number,
+    noOfPeople: Number,
+    noOfBedroom: Number,
+    noOfBathroom: Number
+
+});
+
+
+function validate(property) {
+    console.log({ property })
```
Comment (mohidyousaf) [inline models/propertyInfo.js:25]: remove console

File: routes/propertyInfo.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        return res.status(200).send(properties);
+    } catch (e) {
+        return res.send(e);
+    }
+});
+//Add property
+router.post("/addProperty", async (req, res, next) => {
+    try {
+        //validating the schema
+        console.log({ body: req.body })
```
Comment (mohidyousaf) [inline routes/propertyInfo.js:27]: remove console

**Review by mohidyousaf — COMMENTED** [review]
remove consoles, and check the design i have sent you

File: routes/propertyInfo.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+const moment = require("moment");
+const c = require("config");
+
+//get Property by id the id is yet _id and its listings
+router.get('/:id', async (req, res) => {
+    try {
+        const propertyId = req.params.id;
+
+        const associatedListings = await listings.find({ property: propertyId });
+        const property = await propertyInfo.findById(propertyId);
```
Comment (mohidyousaf) [inline routes/propertyInfo.js:14]: search 'populate' in mongoose, it will help you to populate listing in property return, it is a better way.

**Review by mohidyousaf — APPROVED** [review]
good work. just study populate as well.

File: routes/propertyInfo.js (hunk 3 above)
Comment (prochessor) [reply routes/propertyInfo.js:14]: i saw it at that time i found this a little simpler inshaAllah will update it in the end

**Pull request merged**